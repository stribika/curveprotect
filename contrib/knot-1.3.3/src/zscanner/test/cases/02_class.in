$ORIGIN	.
$TTL	1

; OK
@	IN	NS	@	; The only accepted class IN
@	in	NS	@	; Class in lower-case

; KO
@	CH	NS	@	; Unsupported class
@	CLASS1	NS	@	; Unsupported notation
