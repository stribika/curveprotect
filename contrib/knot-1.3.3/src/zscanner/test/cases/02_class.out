OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=00
------
WARNG=ZSCANNER_EUNSUPPORTED_TYPE
------
WARNG=ZSCANNER_EUNSUPPORTED_TYPE
------
